### ISSUE TITLE
Bootstrap RTL and LTR at the same time

### ISSUE THREAD

**Issue by mohammadreza99** [MohammadYounes/rtlcss#314]
Please someone explain to me that if we use the solution that Bootstrap explained in [documentation](https://getbootstrap.com/docs/5.0/getting-started/rtl/#ltr-and-rtl-at-the-same-time) to use RTL and LTR at the same time, is it supposed to generate a new file for RTL version ? Or do all these events happen in one file?
If a new file is generated, this file is also available for download on the Bootstrap site itself.

**Comment by SheikhRashed** [comment]
did you find any solution in one file?  I'm still searching

**Comment by mohammadreza99** [comment]
@SheikhRashed No, I didn't find it, after a lot of searching I also found a solution to use separate Bootstrap files for this purpose.
